tagstory/src/main/java/no/tagstory/utils/Database.java

```diff
@@ -147,4 +194,63 @@ public boolean insertStatistic(String storyId, Date startTime, long duration, in
 		values.put(STATISTICS_DISTANCE, distance);
 		return db.insert(STATISTICS_TABLE_NAME, null, values) != -1;
 	}
+
+	public boolean insertSaveAndQuit(String statisticId, String historyRootId) {
+		ContentValues mValues = new ContentValues(2);
+		mValues.put(SAVE_QUIT_STATISTIC_ID, statisticId);
+		mValues.put(SAVE_QUIT_HISTORY_ROOT_ID, historyRootId);
+		return db.insert(SAVE_QUIT_TABLE_NAME, null, mValues) != -1;
+	}
+
+	public Cursor getSaveAndQuit(String historyRootId) {
+		return db.query(
+				SAVE_QUIT_TABLE_NAME, new String[] {SAVE_QUIT_ID},
+				SAVE_QUIT_HISTORY_ROOT_ID + "=?", new String[]{historyRootId},
+				null, null, null);
+	}
+
+	public boolean deleteSaveAndQuit(String sqId) {
+		int result = db.delete(SAVE_QUIT_TABLE_NAME, SAVE_QUIT_HISTORY_ROOT_ID
```

Comment:
bedre å ha argument to på neste linje, slik at man kan lese hele argumentet i ett.